bigbluebutton-html5/imports/ui/components/whiteboard/whiteboard-overlay/cursor-listener/component.jsx

```diff
@@ -128,6 +129,12 @@ export default class CursorListener extends Component {
     this.checkCursor(-1, -1);
   }
 
+  clearTouchEvents() {
+    this.removeTouchListeners();
+    this.touchStarted = false;
+    this.handleMouseLeave();
```

Comment:
I'm not really a fan of calling an event listener manually like this. If someone comes later and wants to modify the MouseLeave listener it could cause issues.

Reply:
My issue was with the `this.handleMouseLeave()` usage not the fact that you had a common function clear the touch events.

When the touch ends we want to specifically send (-1,-1) not do whatever `handleMouseLeave()` does.